router/router_system.go

```diff
@@ -103,15 +102,7 @@ func postUpdateConfiguration(c *gin.Context) {
 	if err := c.BindJSON(&cfg); err != nil {
 		return
 	}
-	// Keep the SSL certificates the same since the Panel will send through Lets Encrypt
-	// default locations. However, if we picked a different location manually we don't
-	// want to override that.
-	//
-	// If you pass through manual locations in the API call this logic will be skipped.
-	if strings.HasPrefix(cfg.Api.Ssl.KeyFile, "/etc/letsencrypt/live/") {
```

Comment:
This code should not have been removed, the comment explains what it was doing, the only change that should have happened was removing the to lower call.

Without this every config update will overwrite any custom SSL configuration the user might have added.